bin/reference_data.py

```diff
@@ -453,6 +484,9 @@ def main():
             try:
                 d = downloader(args.species, args.genome_version, args.upload)
                 d.run()
+                # Some species are in both the Ensembl and EnsemblGenome ftp site
+                # break when one is successful
+                break
```

Comment:
Why must you break? What is being broken?

Reply:
This breaks out of the `for` loop at line 483, so that if `EnsemblDownloader` runs successfully, it won't then try and run `EnsemblGenomeDownloader`

Reply:
Cheers. GitHub didn't display that line in the diff view, so I got confused!